docroot/modules/custom/foia_webform/src/FileFieldLookupService.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Drupal\foia_webform;
+
+use Drupal\webform\WebformInterface;
+
+/**
+ * Class FileFieldLookupService.
+ */
+class FileFieldLookupService {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFileAttachmentElementsOnWebform(WebformInterface $webform) {
```

Comment:
If we want to keep this, it seems better if it were protected rather than public.